pkg/cmd/get/environments.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright © 2021 The LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/litmuschaos/litmusctl/pkg/apis/environment"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var GetChaosEnvironmentsCmd = &cobra.Command{
+	Use:   "chaos-environments",
+	Short: "Get Chaos Environments within the project",
+	Long:  `Display the Chaos Environments within the project with the targeted id `,
+	Run: func(cmd *cobra.Command, args []string) {
+		credentials, err := utils.GetCredentials(cmd)
+		utils.PrintError(err)
+
+		projectID, err := cmd.Flags().GetString("project-id")
+		utils.PrintError(err)
+
+		if projectID == "" {
+			utils.White_B.Print("\nEnter the Project ID: ")
+			fmt.Scanln(&projectID)
+
+			for projectID == "" {
+				utils.Red.Println("⛔ Project ID can't be empty!!")
+				os.Exit(1)
+			}
+		}
+
+		environmentList, err := environment.ListChaosEnvironments(projectID, credentials)
```

Comment:
Shouldn't we have a check here to see if `environmentID` is given or not before sending the query